hystrix/metric_collector/metric_collector.go

```diff
@@ -45,6 +45,8 @@ func (m *metricCollectorRegistry) Register(initMetricCollector func(string) Metr
 type MetricCollector interface {
 	// IncrementAttempts increments the number of updates.
 	IncrementAttempts()
+	// IncrementQueueSize increments the number of elements in the queue.
+	IncrementQueueSize()
```

Comment:
This would break any external implementations of this interface. Should probably start managing releases as suggested by https://github.com/afex/hystrix-go/issues/70 before merging this